spk/restic-rest-server/Makefile

```diff
@@ -0,0 +1,30 @@
+SPK_NAME = restic-rest-server
+SPK_VERS = 0.12.1
+SPK_REV = 1
+SPK_ICON = src/logo.png
+DSM_UI_DIR = app
+
```

Comment:
A `DSM_UI_DIR` is not expected when `NO_SERVICE_SHORTCUT = yes` is configured

The `DSM_UI_DIR` is to provide a config file for short cuts in DSM UI and the required icons.
```suggestion

```

Reply:
fixed